### PR TITLE
Swipe action in skills

### PR DESCRIPTION
**Pull request by manthanchauhan**

### CODE REVIEW

File: online_portfolio/static/portfolio/css/portfolio_edit.css [hunk 1]
```diff
@@ -233,6 +233,7 @@
   background-color: rgba(236, 236, 236, 0.86);
   border-radius: 20px 20px 0 0;
   padding: 25px 25px 0 25px;
+  /* border:1px solid palevioletred; */
```
Comment (manthanchauhan) [inline online_portfolio/static/portfolio/css/portfolio_edit.css:237]: @HarshitaAgarwal09 remove comments, keep code clean.

File: online_portfolio/static/portfolio/css/portfolio_edit.css [hunk 2]
```diff
@@ -304,24 +302,37 @@
...
   width: 100%;
 }
 
 .skillButton{
-  margin-top: 75px;
-  background-color: transparent;
-  height: 67vh;
-  outline: none;
-  border: none;
+  /* background-color: transparent; */
```
Comment (manthanchauhan) [inline online_portfolio/static/portfolio/css/portfolio_edit.css:311]: @HarshitaAgarwal09 Remove all the commented code.

File: online_portfolio/static/portfolio/js/edit.js [hunk 3]
```diff
@@ -1,181 +1,181 @@
 function getCookie(name) {
-  var cookieValue = null;
```
Comment (manthanchauhan) [inline online_portfolio/static/portfolio/js/edit.js:2]: @HarshitaAgarwal09 remove whatever formatter you are using..

File: online_portfolio/static/portfolio/js/responsive.js [hunk 4]
```diff
@@ -0,0 +1,55 @@
+console.log("respinsive.js")
+
+var addE = function () {
+    // console.log("add E function called!!")
```
Comment (manthanchauhan) [inline online_portfolio/static/portfolio/js/responsive.js]: @HarshitaAgarwal09 maintain clean code..

File: online_portfolio/static/portfolio/css/portfolio_edit.css [hunk 5]
```diff
@@ -2,6 +2,7 @@
  padding:0px;
  margin:0px;
  /* overflow:hidden; */
+ overflow-x:hidden;
```
Comment (manthanchauhan) [inline online_portfolio/static/portfolio/css/portfolio_edit.css:5]: @HarshitaAgarwal09 remove this, this is causing the scrollbar issue.

File: online_portfolio/static/portfolio/js/responsive.js (hunk 4 above)
Comment (HarshitaAgarwal09) [reply online_portfolio/static/portfolio/js/responsive.js]: I have already removed this part in the code pushed yesterday.

**Pull request merged**